tests/unit_tests/GetVariantInfoTest.php

```diff
@@ -143,7 +143,7 @@ public static function dataProviderGetVariantInfo ()
                 'type' => 'subst',
                 'warnings' => array(),
                 'errors' => array(
-                    'EPOSITIONFORMAT' => 'Too many positions are given for variant type substitution.'
+                    'ETOOMANYPOSITIONS' => 'Too many positions are given for variant type substitution.'
```

Comment:
By giving these cases their own unique error ID, fixHGVS can easily spot and repair them.